tests/torch/graph/test_pattern_parse_from_config.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+import pytest
+
+from nncf.torch.graph.patterns import PATTERN_FACTORY
+from nncf.common.graph.graph_matching import find_subgraphs_matching_expression
+from nncf.torch.graph.patterns import create_graph_pattern_from_pattern_view
+
+
+def create_ref_graph(pattern_view: List[str]):
+    graph = create_graph_pattern_from_pattern_view(pattern_view).graph
+    for node in graph.nodes:
+        graph.nodes[node]["type"] = graph.nodes[node]["type"][0]
+    return graph
+
+
+TEST_PATTERNS_CONFIG = [
+    [
+        [
+            "1 type=[A, B]",
```

Comment:
Does it mean that multiple types are possible for the first node?
Would it be possible to use this syntax in config within `quantize_patterns` parameter?

Reply:
I decided to remove this format until further discussion. If you think that it would be convenient to have it, please report me. Thanks!